lib/price-import.coffee

```diff
@@ -134,6 +137,15 @@ class PriceImport extends ProductImport
     debug 'About to send %s requests', _.size(posts)
     Promise.settle(posts)
 
+  ###
+   * returns true if product does not have any staged changes
+   * and is published
+  ###
+  _canBePublished: (product) ->
+    if !product.hasStagedChanges and product.published
```

Comment:
in the case, if the product has never been published, it needs to be manually approved and then published from the MC.
This feature is for keeping the published products with (price updates) published, as price update does not need approval.